Build is failing with pytest==5.0.0

This is probably from 
https://docs.pytest.org/en/latest/changelog.html#pytest-5-0-0-2019-06-28
```
#5412: ExceptionInfo objects (returned by pytest.raises) now have the same str representation as repr, which avoids some confusion when users use print(e) to inspect the object.
```

Build failure:
```
________________________ test_create_span_with_bad_kind ________________________
    def test_create_span_with_bad_kind():
        with pytest.raises(ZipkinError) as e:
            Span(
                trace_id=generate_random_64bit_string(),
                name='test span',
                parent_id=generate_random_64bit_string(),
                span_id=generate_random_64bit_string(),
                kind='client',
                timestamp=26.0,
                duration=4.0,
            )
    
>       assert 'Invalid kind value client. Must be of type Kind.' in str(e)
E       AssertionError: assert 'Invalid kind value client. Must be of type Kind.' in '<ExceptionInfo ZipkinError tblen=2>'
E        +  where '<ExceptionInfo ZipkinError tblen=2>' = str(<ExceptionInfo ZipkinError tblen=2>)
tests/encoding/_helpers_test.py:25: AssertionError
___________________ test_create_span_with_bad_local_endpoint ___________________
    def test_create_span_with_bad_local_endpoint():
        with pytest.raises(ZipkinError) as e:
            Span(
                trace_id=generate_random_64bit_string(),
                name='test span',
                parent_id=generate_random_64bit_string(),
                span_id=generate_random_64bit_string(),
                kind=Kind.CLIENT,
                timestamp=26.0,
                duration=4.0,
                local_endpoint='my_service',
            )
    
>       assert 'Invalid local_endpoint value. Must be of type Endpoint.' in str(e)
E       AssertionError: assert 'Invalid local_endpoint value. Must be of type Endpoint.' in '<ExceptionInfo ZipkinError tblen=2>'
E        +  where '<ExceptionInfo ZipkinError tblen=2>' = str(<ExceptionInfo ZipkinError tblen=2>)
tests/encoding/_helpers_test.py:41: AssertionError
__________________ test_create_span_with_bad_remote_endpoint ___________________
    def test_create_span_with_bad_remote_endpoint():
        with pytest.raises(ZipkinError) as e:
            Span(
                trace_id=generate_random_64bit_string(),
                name='test span',
                parent_id=generate_random_64bit_string(),
                span_id=generate_random_64bit_string(),
                kind=Kind.CLIENT,
                timestamp=26.0,
                duration=4.0,
                remote_endpoint='my_service',
            )
    
>       assert 'Invalid remote_endpoint value. Must be of type Endpoint.' in str(e)
E       AssertionError: assert 'Invalid remote_endpoint value. Must be of type Endpoint.' in '<ExceptionInfo ZipkinError tblen=2>'
E        +  where '<ExceptionInfo ZipkinError tblen=2>' = str(<ExceptionInfo ZipkinError tblen=2>)
tests/encoding/_helpers_test.py:57: AssertionError
```

--- a/tests/encoding/_helpers_test.py
+++ b/tests/encoding/_helpers_test.py
@@ -22,7 +22,9 @@ def test_create_span_with_bad_kind():
             duration=4.0,
         )
 
-    assert 'Invalid kind value client. Must be of type Kind.' in str(e)
+    assert (
+        'Invalid kind value client. Must be of type Kind.' in str(e.value)
+    )
 
 
 def test_create_span_with_bad_local_endpoint():
@@ -38,7 +40,9 @@ def test_create_span_with_bad_local_endpoint():
             local_endpoint='my_service',
         )
 
-    assert 'Invalid local_endpoint value. Must be of type Endpoint.' in str(e)
+    assert (
+        'Invalid local_endpoint value. Must be of type Endpoint.' in str(e.value)
+    )
 
 
 def test_create_span_with_bad_remote_endpoint():
@@ -54,7 +58,9 @@ def test_create_span_with_bad_remote_endpoint():
             remote_endpoint='my_service',
         )
 
-    assert 'Invalid remote_endpoint value. Must be of type Endpoint.' in str(e)
+    assert (
+        'Invalid remote_endpoint value. Must be of type Endpoint.' in str(e.value)
+    )
 
 
 @mock.patch('socket.gethostbyname', autospec=True)

